packages/algob/sample-project/js/test/sample-test.js

```diff
@@ -14,6 +14,8 @@ describe("Sample Test", function () {
 	let runtime;
 	let lsig;
 	const feeCheckProgram = getProgram("fee-check.teal");
+	//Use getProgramFromPath to getProgram from another path instead of assets
+	//const feeCheckProgram = getProgramFromPath("fee-check.teal", "../assets");
```

Comment:
```suggestion
	// Use getProgramFromPath to getProgram from another path file instead of assets
	// const feeCheckProgram = getProgramFromPath("fee-check.teal", "../assets");
```